SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -55,7 +56,23 @@ void USpatialPlayerSpawner::ReceivePlayerSpawnRequest(Schema_Object* Payload, co
 			URLString += TEXT("?simulatedPlayer=1");
 		}
 		
-		NetDriver->AcceptNewPlayer(FURL(nullptr, *URLString, TRAVEL_Absolute), UniqueId, OnlinePlatformName);
+		FURL Url = FURL(nullptr, *URLString, TRAVEL_Absolute);
+
```

Comment:
I think this would be easier to follow if we just have a conditional for if load-balancing is enabled or not - if not, old flow calling net driver direct, otherwise, all this other logic?